src/ButtonBar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Link } from "@reach/router";
+
+const ButtonBar = () => {
```

Comment:
I think there's a navigation bar in App.js in master which has the same functionality, perhaps pull from master/work with Joey to decide how to merge these two together

Reply:
Merged with updated project in latest commit